### PR TITLE
deploy: docker: prepare 4.8 manifests

### PR DESCRIPTION
**Pull request by fromanirh**
Signed-off-by: Francesco Romani <fromani@redhat.com>

### CODE REVIEW

File: openshift-ci/Dockerfile.bundle-4.7.ci [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# see: https://github.com/operator-framework/operator-registry/blob/master/docs/design/operator-bundle.md
+FROM scratch
+
+ARG CHANNEL=4.7
```
Comment (fedepaol) [inline openshift-ci/Dockerfile.bundle-4.7.ci:4]: Do you want to promote the main dockerfile to 4.8? Or is it intended to be done in a separate PR?

**Comment by fedepaol** [comment]
LGTM assuming the manifests are a copy paste of the 4.7 replacing it with 4.8

File: openshift-ci/Dockerfile.bundle-4.7.ci (hunk 1 above)
Comment (fromanirh) [reply openshift-ci/Dockerfile.bundle-4.7.ci:4]: is meant to be on a separate PR. I'm trying to always keep CI happy, so first adding manifests we don't consume yet, then fixing openshift/release and lastly changing again PAO manifests

**Comment by fromanirh** [comment]
/retest
```
level=fatal msg=failed to fetch Cluster: failed to generate asset "Cluster": failed to create cluster: failed to apply Terraform: failed to complete the change
error: failed to execute wrapped command: exit status 1
```

**Comment by fromanirh** [comment]
/override ci/prow/e2e-gcp
this PR at this stage against master can't break anything

**Comment by fromanirh** [comment]
/override ci/prow/e2e-gcp-operator-upgrade
```
msg=Error: Request "Create IAM Members roles/compute.viewer serviceAccount:ci-op-4hvy7zd7-2a172-65gl6-w@openshift-gce-devel-ci.iam.gserviceaccount.com for \"project \\\"openshift-gce-devel-ci\\\"\"" returned error: Error applying IAM policy for project "openshift-gce-devel-ci": Error setting IAM policy for project "openshift-gce-devel-ci": googleapi: Error 400: The number of members in the policy (1,501) is larger than the maximum allowed size 1,500., badRequest
```
same reasons for https://github.com/openshift-kni/performance-addon-operators/pull/498#issuecomment-762346724 + test passed alread (+ we DO NOT consume these manifests yet...)
